src/pipecat/serializers/twilio.py

```diff
@@ -7,26 +7,33 @@
 import base64
 import json
 
+from pydantic import BaseModel
+
 from pipecat.frames.frames import AudioRawFrame, Frame
 from pipecat.serializers.base_serializer import FrameSerializer
-from pipecat.utils.audio import ulaw_8000_to_pcm_16000, pcm_16000_to_ulaw_8000
+from pipecat.utils.audio import ulaw_to_pcm, pcm_to_ulaw
 
 
 class TwilioFrameSerializer(FrameSerializer):
+    class InputParams(BaseModel):
+        twilio_sample_rate: int = 8000
+        sample_rate: int = 16000
+
     SERIALIZABLE_TYPES = {
         AudioRawFrame: "audio",
     }
 
-    def __init__(self, stream_sid: str):
+    def __init__(self, stream_sid: str, params: InputParams = InputParams()):
         self._stream_sid = stream_sid
+        self._params = params
 
     def serialize(self, frame: Frame) -> str | bytes | None:
         if not isinstance(frame, AudioRawFrame):
             return None
 
         data = frame.audio
 
-        serialized_data = pcm_16000_to_ulaw_8000(data)
+        serialized_data = pcm_to_ulaw(data, frame.sample_rate, self._params.twilio_sample_rate)
```

Comment:
Hi, this should also be changed to `self._params.sample_rate` without it the issue persists.